apis/v1alpha2/httproute_types.go

```diff
@@ -762,10 +762,15 @@ type HTTPRouteForwardTo struct {
 	// The gateway status for this route should be updated with a
 	// condition that describes the error more specifically.
 	//
+	// If there is a cross-namespace reference to an *existing* object
+	// with no ReferencePolicy, the controller must ensure the "ResolvedRefs"
+	// condition on the Gateway is set to `status: true`, with the "RefNotPermitted"
```

Comment:
I think `ResolvedRefs` would be false in this case? i.e the reference could not be resolved because there was no policy that permitted access.

Reply:
🤦 Yes, thanks.